File: JobServiceEx/app/src/main/java/com/example/jobserviceex/MainActivity.java
package com.example.jobserviceex;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.jobserviceex.notify.NotificationManager;

public class MainActivity extends AppCompatActivity {
	public static final String NOTIFICATION_CHANNEL_ID = "ex_notify_channel";

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_main);
		NotificationManager.scheduleJob(getApplicationContext());
	}
}

File: DemoPDFView/app/src/main/java/com/example/demopdfview/MainActivity.kt
package com.example.demopdfview

import android.net.Uri
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.pspdfkit.configuration.activity.PdfActivityConfiguration
import com.pspdfkit.datastructures.TextSelection
import com.pspdfkit.ui.special_mode.manager.TextSelectionManager

class MainActivity : AppCompatActivity(), TextSelectionManager.OnTextSelectionChangeListener {
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)
		setContentView(R.layout.activity_main)

		val uri = Uri.parse("file:///android_asset/100MauCaGiaoTiepTiengNhat.pdf")
		val config = PdfActivityConfiguration.Builder(this).build()
	}

	override fun onBeforeTextSelectionChange(p0: TextSelection?, p1: TextSelection?): Boolean {
		TODO("Not yet implemented")
	}

	override fun onAfterTextSelectionChange(p0: TextSelection?, p1: TextSelection?) {
		TODO("Not yet implemented")
	}
	
}